README.md

```diff
@@ -76,4 +77,16 @@ By default, installing a package with the `—save` option, npm saves the packag
 npm config set save-exact
 ```
 
+[↑ back to Table of Contents](#table-of-contents)
+
+## Write atomic tasks
+
+### Why?
+
+Convoluted tasks are harder to debug and understand. 
```

Comment:
I would turn this around into positive reasons:
* Atomic tasks are easy to read and understand.
* Atomic tasks are easy to reuse.